helm/vitess/templates/_orchestrator-conf.tpl

```diff
@@ -35,13 +35,14 @@ data:
     "DetectClusterDomainQuery": "",
     "DetectInstanceAliasQuery": "SELECT value FROM _vt.local_metadata WHERE name='Alias'",
     "DetectPromotionRuleQuery": "SELECT value FROM _vt.local_metadata WHERE name='PromotionRule'",
+    "DetectDataCenterQuery": "SELECT value FROM _vt.local_metadata WHERE name='DataCenter'",
```

Comment:
Is `DataCenter` hardcoded into Vitess somewhere?

Reply:
https://github.com/vitessio/vitess/blob/master/go/vt/vttablet/tabletmanager/restore.go#L216